transport/nio/pom.xml

```diff
@@ -123,6 +133,34 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
             </plugin>
+<!-- Enable to build benchmarks.jar after there is new version of plugin
```

Comment:
What's the plan for getting this to work? Would be worth adding more comment to explain, and say how it can be run now if folks want to run it (maybe QA).